x-pack/plugins/task_manager/server/config.ts

```diff
@@ -51,6 +51,12 @@ const eventLoopDelaySchema = schema.object({
   }),
 });
 
+const requeueInvalidTasksConfig = schema.object({
+  enabled: schema.boolean({ defaultValue: false }),
+  delay: schema.number({ defaultValue: 3000, min: 0 }),
+  max_attempts: schema.number({ defaultValue: 20, min: 1 }),
```

Comment:
OK made it 100.

Yes this is for ZDT. And probably it will happen after a new Kibana instance pick up the task.
If a new version introduces a new field and adds it to the Rule SO, an old one may fail or corrupt the data. Therefore we run the validator and skip if the data is invalid.